thoth-app/src/component/mod.rs

```diff
@@ -276,7 +300,25 @@ macro_rules! pagination_component {
                                             <tr>
                                                 {
                                                     for self.table_headers.iter().map(|h| {
-                                                        html! {<th>{h}</th>}
+                                                        {
+                                                            // If the header is a sortable field, make it clickable
+                                                            match <$order_field>::from_str(&h) {
+                                                                Ok(header) => {
+                                                                    html! {
+                                                                        <th
```

Comment:
We should add the `is-clickable` class here to make it obvious to the user that clicking on the columns will do something